docker/build.sh

```diff
@@ -21,11 +21,17 @@ function make_jar()
     # Get the last commit hash of the model directory and set it as an environment variable
     GIT_MODEL_HASH=$(git log -n 1 --pretty=format:%h model)
 
-    # make jar.  cache sbt dependencies.
-    docker run --rm --link mysql:mysql -e GIT_MODEL_HASH=$GIT_MODEL_HASH -v $PWD:/working -v jar-cache:/root/.ivy -v jar-cache:/root/.ivy2 broadinstitute/scala-baseimage /working/docker/install.sh /working
+    # make jar.  cache sbt dependencies. capture output and stop db before returning.
+    JAR_CMD=`docker run --rm --link mysql:mysql -e GIT_MODEL_HASH=$GIT_MODEL_HASH -v $PWD:/working -v jar-cache:/root/.ivy -v jar-cache:/root/.ivy2 broadinstitute/scala-baseimage /working/docker/install.sh /working`
```

Comment:
do you mean to set this to a var?

Reply:
it's the best way i know of to capture the exit code of the command without it causing the script to fail (we have the `-x` flag set).  am open to suggestions of how to do it better though